docs/getting-started.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 1.1
+---
+
+# Getting Started
+
+The `@lukso/lsp6-signer.js` package is used to sign a transaction.
```

Comment:
Specifically it can only be used to sign an LSP6 execute relay call transaction. Normal blockchain transactions will not work if signed with this so I think good to specify to give context